app/assets/javascripts/parameter_override.js

```diff
@@ -7,12 +7,15 @@ function override_param(item) {
   var param_value = param.find('[id^=value_]');
   var v = param_value.val();
 
-  $('#parameters')
-    .find('.btn-primary')
-    .click();
-  var new_param = $('#parameters')
-    .find('.fields')
-    .last();
+  var addParameterButton = $('#parameters').find('.btn-primary');
+  addParameterButton.click();
+  var directionOfAddedItems = addParameterButton.attr('direction');
+  var new_param = $('#parameters').find('.fields');
+  if(directionOfAddedItems === 'append'){
+    new_param = new_param.last();
```

Comment:
I didn't find where this case can happen, can you attach a link?

Reply:
This is where the items are added: https://github.com/theforeman/foreman/blob/develop/app/assets/javascripts/application.js#L150